Providers/OpenEBS/Installers/Operator/0.6/ansible/openebs.yaml

```diff
@@ -0,0 +1,36 @@
+# TODO 
+# Apply openebs operator and openebs storage class
+
+- hosts: localhost
+  connection: local  
+  tasks:
+   - name: Applying openebs operator
+     shell: kubectl --kubeconfig /root/admin.conf apply -f https://raw.githubusercontent.com/openebs/openebs/v0.6/k8s/openebs-operator.yaml 
```

Comment:
Just a thought: can we create an alias (ansible var) for the function "kubectl --kubeconfig /root/admin.conf" and use them, say, `kubeapply: "kubectl --kubeconfig /root/admin.conf"`

```
shell: {{ kubeapply }} <artifact>
```

Similarly, the artifact link can be in the vars.